lib/datadome/configuration.rb

```diff
@@ -9,13 +9,13 @@ def initialize
       @api_server = "api.datadome.co"
       @exclude_matchers = []
       @include_matchers = []
-      @monitor_mode = false
+      @intercept_matchers = []
```

Comment:
Do you mind adding some documentation on how to use it?

Reply:
good point

Reply:
done.